pkg/config/config.go

```diff
@@ -841,17 +841,21 @@ func ParseConfiguration(configurationFilePath string, opts ...ConfigurationParsi
 	}
 
 	const (
-		defaultEnvVarHOME   = "/home/build"
-		defaultEnvVarGOPATH = "/home/build/.cache/go"
+		defaultEnvVarHOME       = "/home/build"
+		defaultEnvVarGOPATH     = "/home/build/.cache/go"
+		defaultEnvVarGOMODCACHE = "/var/cache/melange/gomodcache"
 	)
 
-	if cfg.Environment.Environment["HOME"] == "" {
-		cfg.Environment.Environment["HOME"] = defaultEnvVarHOME
-	}
-	if cfg.Environment.Environment["GOPATH"] == "" {
-		cfg.Environment.Environment["GOPATH"] = defaultEnvVarGOPATH
+	var setIfEmpty = func(key, value string) {
+		if cfg.Environment.Environment[key] == "" {
```

Comment:
This may be a pointless idea, but there's a chance it's better to use the `, ok =` style of check for map value emptiness rather than an empty string, in case we think the user would ever want to explicitly control the env variable to be empty at some point.